src/components/account/EditProfile.tsx

```
@@ -0,0 +1,284 @@
+      formData.append("phoneNumber", data.phoneNumber);
+      formData.append("dob", birthDate ? formatDate(birthDate, "dmy") : "");
+      formData.append("file", file || avatarBlob || "");
+      const userJson = localStorage.getItem("user");
+      if (userJson) {
+        const user = JSON.parse(userJson);
+        formData.append("id", user.id);
+        const updateResponse = await userApi.update(user.id, formData);
+        if (updateResponse.status == StatusCode.OK) {
+          notify("Edit profile successed!");
+          setIsEditing(false);
+          var res = await userApi.getById(user.id);
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)